__docs__/wonder-blocks-switch/switch.argtypes.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+
+const iconsMap: Record<string, React.ReactElement<typeof Icon>> = {};
+
+Object.entries(icons).forEach(([iconLabel, iconValue]) => {
+    iconsMap[iconLabel] = <Icon icon={iconValue} />;
+});
+
+export default {
+    ariaLabel: {
+        description: "A label for the switch.",
+        table: {
+            category: "Accessibility",
+            type: {
+                summary: "string",
+                detail: "If a visible label is not provided, aria-label should be used to describe the purpose of the switch.",
+            },
+        },
+    },
+    checked: {
+        description: "Whether this compoonent is checked.",
```

Comment:
```suggestion
        description: "Whether this component is checked.",
```

Reply:
my keyboard has been acting up lately with these O's. i guess it made it through...